### ISSUE TITLE
Let's highlight the double curly braces, like in JSP language

### ISSUE THREAD

**Issue by revelt** [alohaas/language-nunjucks#23]
Hi all,

Currently, Nunjucks variables within double curly braces are not highlighted any way:

![nunjucks_curly](https://cloud.githubusercontent.com/assets/8344688/24711077/d8fff72a-1a16-11e7-9fa2-4bfe8bb8576f.png)
—_HTML(Nunjucks Templates)_ language

How nice would it be if the double curly braces were highlighted, for example, like in JSP language, notice the orange bits:

![jsp_curly](https://cloud.githubusercontent.com/assets/8344688/24711138/fe70a5d6-1a16-11e7-90c3-30aa1c8ab3c2.png)
—_JavaServer Pages_ language

What do you think?

**Comment by alohaas** [comment]
@revelt you're looking to have the curly braces a different color than the variable name? I think that's a nice idea :)

**Comment by revelt** [comment]
This issue is probably blocked by #21, isn't it?

**Comment by frasertmay** [comment]
@revelt Yeah I think once #21 is sorted out then we should be able to do this

**Comment by revelt** [comment]
I'm thinking, practically, should we attract some attention by Atom core team regarding #21 ? At least to get some consultation, _is it possible at all_ and if so, _how_ can this be done? Even a technical evaluation of the task would help to start issue moving eventually.

I'm worried that issues like this require deep expertise and neither commoners (like me) nor maintainers (correct me if I'm wrong) are not capable to solve this by themselves. We passively wait for somebody with expertise to come in and solve this, but statistically it's not likely. More often common people (like me) come in, have a glance but don't touch it because of lack of expertise.

I know Atom core team did work on languages so it might be within their scope.

@frasertmay What do you think?

**Comment by frasertmay** [comment]
It seems the [`language-mustache`](https://github.com/atom/language-mustache) package was written by the Atom core team and they seem to manage this quite well:

![screenshot at 2017-10-27 15 50 51](https://user-images.githubusercontent.com/3003251/32110213-e4680760-bb2e-11e7-954c-c1924116cac1.png)

I'm not sure if they have a solution for #21 but I'll have a look.
